packages/backend-common/src/reading/GithubUrlReader.test.ts

```diff
@@ -230,4 +243,91 @@ describe('GithubUrlReader', () => {
       );
     });
   });
+
+  describe('readTree', () => {
+    it('returns the files from a archive', async () => {
+      const worker = setupServer();
+
+      const repoBuffer = fs.readFileSync(
+        path.resolve(dirname, 'test-data', 'repo.tar.gz'),
+      );
+
+      worker.use(
+        rest.get(
+          'https://github.com/spotify/mock/archive/repo.tar.gz',
+          (_, res, ctx) =>
+            res(
+              ctx.status(200),
+              ctx.set('Content-Type', 'application/x-gzip'),
```

Comment:
Hah, had to Google - TIL, interesting that tar doesn't have a supertype for this